coral/src/app/features/topics/request/hooks/useGetEnvironmentsForTeam.tsx

```diff
@@ -0,0 +1,12 @@
+import { useQuery, UseQueryResult } from "@tanstack/react-query";
+import { Environment } from "src/domain/environment";
+import { getEnvironmentsForTeam } from "src/domain/environment/environment-api";
+
+function useGetEnvironmentsForTeam(): UseQueryResult<Environment[]> {
```

Comment:
Should we apply the policy we defined where we do that kind of query inline instead of in a custom hook?

Reply:
Lets do it!